### ISSUE TITLE
Script de mise à jour en masse de la date de dématérialisation des démarches déjà dématérialisées

### ISSUE THREAD

**Issue by slauriere** [DISIC/wikidemarches#374]
Suite à #364, écrire un script de mise à jour des démarches déjà dématérialisées.

**Comment by slauriere** [comment]
Je me demande si cette tâche est pertinente. L'idée était que, pour les démarches qui sont déjà dématérialisées mais qui n'ont pas de date de dématérialisation, on indique une date dans le passé. Mais on ne pourra connaître réellement cette date que déclarativement, donc pas via un script. Je ferme ce ticket, je vous laisse le rouvrir si besoin.

**Comment by slauriere** [comment]
Ceci dit, pour calculer des stats d'évolution de la dématérialisation, il sera important d'avoir des dates pour toutes les démarches dématérialisées. Est-ce que, pour la démarches déjà dématérialisées mais qui n'ont pas de date de dématérialisation indiquée, on renseigne ce champ automatiquement, par exemple en utilisant la date de premier enregistrement de la démarche sur le site ?

**Comment by georges-bayard** [comment]
2 sujets sous jacents : 
l'historisation des démarches. Pas sur qu'on ait un besoin très fort. On peut résoudre le besoin avec l'idée qu'on gèle une photo mensuelle qu'on avait évoqué
l'historisation des votes : c'est déjà le cas ; tout vote est associé à une date. Après le sujet sera de représenter tout cela
ok pour fermer donc

**Comment by georges-bayard** [comment]
réponse à ton commentaire précédent : on peut le faire "par précaution" mais cela n'a pas grand sens du point de vue métier. Le seul besoin que je vois c'est en 2022 comparer avec la vue 2019

**Comment by slauriere** [comment]
Oui, si on veut comparer l'état de la dématérialisation en 2022 à celui de 2019, il faudra que ces dates soient renseignées. On pourra toujours les renseigner plus tard, en faisant l'hypothèse (réaliste j'imagine) que les démarches qui seront dématérialisées entre temps auront bien un champ date renseigné (sinon ce sera plus compliqué de les distinguer de celles qui étaient déjà dématérialisées en 2019, il me semble).
